packages/sdk/react-client/src/testing/ClientRepeater.tsx

```diff
@@ -36,12 +34,12 @@ export type ClientRepeaterProps<P extends RepeatedComponentProps> = {
  * Utility component for Storybook stories which sets up clients for n peers.
  * The `Component` property is rendered n times, once for each peer.
  */
+// TODO(burdon): To discuss: evolve ClientRepeater with optional decorator that uses it.
```

Comment:
I have no issues with replacing `ClientRepeater` with decorators, assuming they work. The only reason this exists is because the previous attempts at client decorators didn't work, were flaky, etc.

Reply:
right. let's decide how to test this before i start replacing them.
as i remember that was a long time ago, so perhaps our knowledge and the tools themselves have changed since then.